bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -37,55 +37,72 @@ class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
 	enum OpType { OP_INSERT, OP_CLEAR, OP_CLEAR_RANGE, OP_READ, OP_GET_RANGES, OP_LAST = OP_GET_RANGES };
 	std::vector<OpType> excludedOpTypes;
 
+	// Allow reads at the start to get blob_granule_transaction_too_old if BG data isn't initialized yet
+	// FIXME: should still guarantee a read succeeds eventually somehow
```

Comment:
Because the blob granule data is initialized after the rest of the database, so it is possible for the database to be readable at version X, but the blob granule data hasn't been created yet, so version X is not readable from blob granules.
I also handle this issue in the BlobGranuleVerifier and BlobGranuleCorrectness* simulation workloads, but forgot to handle it here.